test/e2e/operations.go

```diff
@@ -0,0 +1,87 @@
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/Azure/go-autorest/autorest/to"
+
+	"github.com/Azure/ARO-RP/pkg/client/services/preview/redhatopenshift/mgmt/2019-12-31-preview/redhatopenshift"
+)
+
+var _ = Describe("List operations", func() {
+	Specify("the correct static operations are returned", func() {
+		ctx := context.Background()
+
+		opList, err := Clients.Operations.List(ctx)
+		Expect(err).NotTo(HaveOccurred())
+
+		l := &redhatopenshift.OperationList{
```

Comment:
feels a bit brittle, I think testing len(l.Operations) > 0 would probably be ok.  If we want to test the detail a unit test would make sense.